newIDE/app/src/ObjectGroupsList/index.js

```diff
@@ -67,430 +80,524 @@ type Props = {|
-                    height={height}
-                    getItemName={getGroupWithContextName}
-                    getItemId={(groupWithContext, index) => {
-                      return 'group-item-' + index;
-                    }}
-                    isItemBold={isGroupWithContextGlobal}
-                    onEditItem={groupWithContext =>
-                      this.props.onEditGroup(groupWithContext.group)
-                    }
-                    selectedItems={
-                      this.state.selectedGroupWithContext
-                        ? [this.state.selectedGroupWithContext]
-                        : []
-                    }
-                    onItemSelected={this._selectGroup}
-                    renamedItem={renamedGroupWithContext}
-                    onRename={this._onRename}
-                    buildMenuTemplate={this._renderGroupMenuTemplate(i18n)}
-                    onMoveSelectionToItem={this._moveSelectionTo}
-                    canMoveSelectionToItem={this._canMoveSelectionTo}
-                    reactDndType={groupWithContextReactDndType}
-                  />
-                )}
-              </I18n>
-            )}
-          </AutoSizer>
+          <I18n>
+            {({ i18n }) => {
+              const treeViewData = getTreeViewData(i18n);
+              const globalRootItem = treeViewData[0];
+              const initiallyOpenedNodeIds = [
+                globalRootItem.isRoot &&
+                globalRootItem.children.length === 1 &&
+                'isPlaceholder' in globalRootItem.children[0]
+                  ? null
+                  : globalGroupsRootFolderId,
+                sceneGroupsRootFolderId,
+              ].filter(Boolean);
+
+              return (
+                <div style={{ flex: 1 }}>
+                  <AutoSizer style={{ width: '100%' }} disableWidth>
+                    {({ height }) => (
+                      <TreeView
+                        key={listKey}
+                        ref={treeViewRef}
+                        items={treeViewData}
+                        height={height}
+                        searchText={searchText}
+                        getItemName={getTreeViewItemName}
+                        getItemChildren={getTreeViewItemChildren}
+                        multiSelect={false}
+                        getItemId={getTreeViewItemId}
+                        onEditItem={editItem}
+                        selectedItems={
+                          selectedGroupWithContext
+                            ? [selectedGroupWithContext]
+                            : []
+                        }
+                        onSelectItems={items => {
+                          if (!items) setSelectedGroupWithContext(null);
+                          const itemToSelect = items[0];
+                          if (
+                            'isRoot' in itemToSelect ||
+                            'isPlaceholder' in itemToSelect
```

Comment:
those strings are a bit magic, should they be in a constant somewhere? (you're using it in multiple places too)